flexible/pubsub/README.md

```diff
@@ -0,0 +1,58 @@
+# App Engine Flexible Environment - Pub/Sub Sample
+
+## Clone the sample app
+
+Copy the sample apps to your local machine, and cd to the pubsub directory:
+
+```
+git clone https://github.com/GoogleCloudPlatform/java-docs-samples
+cd java-docs-samples/flexible/pubsub
+```
+
+## Setup
+
+Make sure `gcloud` is installed and authenticated.
+
+Create a topic
+```
+gcloud beta pubsub topics create <your-topic-name>
+```
+
+Create a subscription, which includes specifying the endpoint to which the Pub/Sub server should send requests.
```

Comment:
Maybe change this to: 

Create a [push subscription](https://cloud.google.com/pubsub/docs/push), to send pushed messages to a Google Cloud Project URL such as `https://<your-project-id>.appspot.com/`.